hubgrep/frontend_blueprint/routes/search.py

```diff
@@ -32,8 +32,7 @@ def search():
     if form.search_phrase:
         terms = form.search_phrase.split()
         search_interfaces = get_hosting_service_interfaces()
-        results, failed_responses = fetch_concurrently(terms, search_interfaces)
-        external_errors = [response.error_msg for response in failed_responses]
+        results, external_errors = fetch_concurrently(terms, search_interfaces)
```

Comment:
each external_error is a tuple of (hosting_service_interface, cached_response) now.
still not nice, but all the info is there.